ci-operator/step-registry/cucushift/installer/rehearse/azure/cases/multi-clusters/same-clustername-diff-basedomain/deprovision/cucushift-installer-rehearse-azure-cases-multi-clusters-same-clustername-diff-basedomain-deprovision-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: cucushift-installer-rehearse-azure-cases-multi-clusters-same-clustername-diff-basedomain-deprovision
+  from: upi-installer
```

Comment:
I guess we can use `installer` image here.

Reply:
yes, updated, also updated cucushift-installer-rehearse-azure-cases-multi-clusters-same-subnets-deprovision-ref.yaml together.